gfx.cpp

```diff
@@ -8921,7 +8928,7 @@ class GfxInternal
         if(debug_shaders_)
         {
             shader_args.push_back(DXC_ARG_DEBUG);
-            shader_args.push_back(DXC_ARG_OPTIMIZATION_LEVEL0);
+            shader_args.push_back(DXC_ARG_SKIP_OPTIMIZATIONS);
```

Comment:
Out of curiosity, what's the difference between `DXC_ARG_OPTIMIZATION_LEVEL0` and `DXC_ARG_SKIP_OPTIMIZATIONS`?

Reply:
O0 performs minor optimisations and Od doesnt do anything. Od can result in some terribly performing code compilation compared to O0 but O0 can also optimise away some variables such that they cant be viewed in PIX

Reply:
Unfortunately microsoft doesnt really provide much information on what exactly is difference ([https://learn.microsoft.com/en-us/windows/win32/direct3dhlsl/d3dcompile-constants](https://learn.microsoft.com/en-us/windows/win32/direct3dhlsl/d3dcompile-constants))
Od is less likely to cause compiler crashes though in my testing